kotest-core/src/jvmMain/kotlin/io/kotest/core/engine/SpecRunner.kt

```diff
@@ -47,9 +51,7 @@ abstract class SpecRunner(val listener: TestEngineListener) {
 
       val futures = testCases.map { testCase ->
          executor.submit {
-            runBlocking {
```

Comment:
I'm not sure I understand the error in the `ubuntu` build since it failed with.

> e: /home/runner/work/kotest/kotest/kotest-core/src/jvmMain/kotlin/io/kotest/core/engine/SpecRunner.kt: (52, 13): Unresolved reference: runBlocking
e: /home/runner/work/kotest/kotest/kotest-core/src/jvmMain/kotlin/io/kotest/core/engine/SpecRunner.kt: (53, 16): Suspension functions can be called only within coroutine body

But the relevant code is removed in this PR. Do I have a missunderstanding of how CI checks run in Kotest?